modules/backup-and-restore/pages/differential-backups.adoc

```diff
@@ -0,0 +1,91 @@
+
+* Differential backups comprise data files that have changed since the most recently completed full backup.
+* Incremental backups involve data files changed since the last incremental backup.
+
+For example, a full backup happened at `tid:100`, a schema change happened at `tid:150` and an differential backup happened at `tid:200`.
+
+== How Differential Backup works
+
+To illustrate how differential backup works, take into account the user scenario below.
+
+.User Scenario
+====
+. A user creates a full backup every __Sunday__ (Or Day `N`).
+
+. During the week, between the two full backup days (__Sunday__ and __Next Sunday__) TigerGraph provides differential backups for every change since that first full backup.
+
+. A differential backup is taken until the backup data is as large as the backup on __Sunday__
+. Another full backup is taken on __Next Sunday__ and the circle repeats steps 2-4.
+====
+
+=== Enable Differential Backup
+
+If a user wants to make differential backups based on a full backup they have made, they should run:
+`gadmin backup create <tag>` with the `--incremental` flag.
+
+.Ex.
+[console, gsql]
+----
+gadmin backup create <tag> --incremental
+----
+
+[NOTE]
+====
+For more details on `gadmin backup create` see xref:tigergraph-server:backup-and-restore:backup-cluster.adoc#_procedure[Back up a Database Cluster Procedure].
+====
+
+To restore a differential backup, users need to find the correct `<tag>` from the backup list.
+To see the list of backups, run:
+
+.Ex.
+[console, gsql]
+----
+gadmin backup list
+----
+
+.`gadmin backup list` example result.
+image::backup_list.png[]
+
+This will list all the backup tags. Notice the type column in the result. This is how users can identify which backups are `FULL` or not.
+To restore the specific backup, run:
+
+.Ex.
+[console, gsql]
+----
+gadmin backup restore <specifc backup tag>
+----
+
+=== Disable Differential Backup
+
+If a user wants to make a full backup _**WITHOUT**_ differential backup enabled, they can do so by excluding the `--incremental` flag in `gadmin backup create`.
+
+.Ex.
+[console, gsql]
+----
+gadmin backup create <tag>
+----
+
+== Limitations
+
+Differential backup does have some know limitations:
+
+* Currently, schema change is not yet supported.
+This means users cannot run a differential backup, if they have run schema changes since last full snapshot backup.
+Users have to run a full snapshot backup first, then they can use differential Backup.
+
+* Currently, restored TigerGraph is not supported yet.
+If the TigerGraph was restored from a full snapshot backup, users cannot run a differential backup.
+Users have to run a full snapshot backup first, then they can use Differential Backup.
+
+* Open transactions at the full backup happening is not yet supported.
+This may cause data lost.
+
+* Kafka reset or kafka purging too often is not yet supported by differential backups.
```

Comment:
"often" might not be too accurate here. For purging, the amount of kafka messages purged matters, not the frequency of Kafka purging.

Reply:
Thank you changed this with latest commit.